### PR TITLE
fix broken test app moderatorfeedback

### PR DESCRIPTION
**Pull request by goapunk**
The moderatorfeedback was missing the `__init__.py` which causes errors on django 3.x and the initial migration (since the migration in `tests/apps/ideas` depends on that (not sure whether we need that dependency?).

### CODE REVIEW

File: tests/apps/moderatorfeedback/apps.py [hunk 1]
```diff
@@ -3,4 +3,4 @@
 
 class Config(AppConfig):
     name = 'tests.apps.moderatorfeedback'
-    label = 'a4test_moderatorfeedback'
+    label = 'moderatorfeedback'
```
Comment (fuzzylogic2000) [inline tests/apps/moderatorfeedback/apps.py:6]: Why did you change this?

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!

File: tests/apps/moderatorfeedback/apps.py (hunk 1 above)
Comment (goapunk) [reply tests/apps/moderatorfeedback/apps.py]: because the migrations would fail otherwise on Django 3.x. Weirdly it now fails on django 2.x

**Comment by goapunk** [comment]
Ok, apparently this doesn't work in Django 2.x anymore, so I will just close this as the reason I fixed this was compatibility with Django 3.x

**Comment by fuzzylogic2000** [comment]
Nooo! I have to decide between the tiny inconsistency in the app name and having a migration here, I would go for the migration... :p

**Comment by goapunk** [comment]
Yes, but weirdly this migration is working in Django 2.x even if it shouldn't so I guess it's fine ? :)
